.circleci/config.yml

```
@@ -75,6 +75,20 @@ jobs:
       - run: make pristine
       - run: make test
 
+  test_windows:
+    machine:
+      image: windows-default
```

```suggestion
    executor: win/default
```

I have literally no experience with it but reading CircleCI docs, the machine image isn't available in the Cloud version of CircleCI.

What if we used the `win/defaut` executor as documented?

https://circleci.com/docs/2.0/executor-types/#using-the-windows-executor

---

let's try, I have no experience with CircleCI either :)

---

@mumoshu it did not work, from what I searched, I think there is a toggle flag on Circle CI side to activate the ability to use windows runner